kategory-test/src/main/kotlin/kategory/laws/TraverseFilterLaws.kt

```diff
@@ -0,0 +1,21 @@
+package kategory.laws
+
+import io.kotlintest.properties.forAll
+import kategory.*
+
+object TraverseFilterLaws {
+
+    inline fun <reified F> laws(TF: TraverseFilter<F> = traverseFilter<F>(), GA: Applicative<F> = applicative<F>(), crossinline cf: (Int) -> HK<F, Int>, EQ: Eq<HK<F, Int>>): List<Law> =
+            TraverseLaws.laws(TF, GA, cf, EQ) + FunctorLaws.laws(GA, cf, EQ) + listOf(
```

Comment:
The traverse laws already include the functor laws so no need to add them here again or tests will run twice